rts/Sim/Misc/NanoPieceCache.h

```diff
@@ -21,7 +22,7 @@ struct NanoPieceCache
 
 	void Update() { curBuildPowerMask >>= 1; }
 
-	float GetBuildPower() const { return (count_bits_set(curBuildPowerMask) / float(UNIT_SLOWUPDATE_RATE)); }
+	float GetBuildPower() const { return (count_bits_set(curBuildPowerMask) / float(MASK_BITS)); }
```

Comment:
`count_bits_set` could be migrated to `https://en.cppreference.com/w/cpp/numeric/popcount`

Reply:
#1331